### ISSUE TITLE
[Bug Report] aria-label use itemperpagetext as is instead of translating it

### ISSUE THREAD

**Issue by axbarbosa** [vuetifyjs/vuetify#12012]
### Environment
**Vuetify Version:** 2.3.8
**Vue Version:** 3.0.0-rc.5
**Browsers:** Firefox 79.0
**OS:** Linux

### Steps to reproduce
Just navigate through the app to the number of rows per page input element and it will read out loud "10 $vuetify.dataTable.itemsPerPageText"
You can also use the accessibility tab in the developer tools (maj+f12)

### Expected Behavior
The itemsPerPageText is a reference to the locale file, the variable should be interpreted so that the aria-label can be understood.

### Actual Behavior
The variable is used as a text.

### Reproduction Link
<a href="https://codepen.io/axbarbosa/pen/gOrbaPr" target="_blank">https://codepen.io/axbarbosa/pen/gOrbaPr</a>

### Other comments
The example is coming straight from the data table documentation page, it's the first one

**Comment by jacekkarczmarczyk** [comment]
https://github.com/vuetifyjs/vuetify/blob/master/packages/vuetify/src/components/VDataIterator/VDataFooter.ts#L116
